rest-api-spec/src/main/resources/rest-api-spec/api/indices.exists.json

```diff
@@ -13,14 +13,6 @@
         }
       },
       "params": {
-        "ignore_unavailable": {
-            "type" : "boolean",
-            "description" : "Whether specified concrete indices should be ignored when unavailable (missing or closed)"
-        },
-        "allow_no_indices": {
-            "type" : "boolean",
-            "description" : "Whether to ignore if a wildcard indices expression resolves into no concrete indices. (This includes `_all` string or when no indices have been specified)"
-        },
```

Comment:
I agree it makes no sense and I'm happy with this fix. I just think that a short line saying "params X and Y have been removed from the Indices Exists REST API" in changelog is still valuable. But as you want.